### PR TITLE
more information on kernel-sbom

### PR DESCRIPTION
**Pull request by deitch**
Adds purl, package version and licenses, where possible, to kernel sbom.

It also adds a comment to the kernel dockerfile so that it does not include it in the sbom, but that part doesn't work yet, due to how our dockerfile parser works. There will be a follow-on for that at some point, but this is good.

### CODE REVIEW

**Comment by deitch** [comment]
Strange. The unit tests failed in places that I did not touch.

File: tools/dockerfile-add-scanner/cli/licenses.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package cli
```
Comment (eriknordmark) [inline tools/dockerfile-add-scanner/cli/licenses.go:1]: Should this file have some Copyright (the original one from where you coped)?
A License SPDX?

Comment (deitch) [reply tools/dockerfile-add-scanner/cli/licenses.go:4]: Yeah, good point. They all should. Just added them.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**